CHANGELOG.md

```diff
@@ -3,6 +3,31 @@
 ## Current Develop Branch
 
 ## 8.1.0 Mon Oct 05 2020
+- [#9422](https://github.com/MetaMask/metamask-extension/pull/9422): Fix data backup feature (i.e. syncing with 3box)
+- [#9434](https://github.com/MetaMask/metamask-extension/pull/9434): Improve gas input UI by using tooltip instead of ugly modal to communicate gas data
```

Comment:
```suggestion
- [#9434](https://github.com/MetaMask/metamask-extension/pull/9434): Improve gas input UI by using tooltip instead of a modal to communicate gas data
```